fastlane/Fastfile

```diff
@@ -184,8 +184,13 @@ platform :ios do
   lane :complete_code_freeze do |options|
     ios_completecodefreeze_prechecks(options)
     generate_strings_file_for_glotpress
-    version = ios_get_app_version
-    trigger_beta_build(branch_to_build: "release/#{version}")
+
+    if ENV.fetch('RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM', false) || UI.confirm('Push changes to the remote and trigger the beta build?')
+      push_to_git_remote(tags: false)
+      trigger_beta_build(branch_to_build: "release/#{ios_get_app_version}")
+    else
+      UI.message('Aborting code freeze completion as requested.')
+    end
```

Comment:
As I mentioned in the PR description, this is consistent with what WordPress iOS does.

How do you feel about it? For me:

- Pros: It's good to have a human explicitly push to the remote.
- Cons: If you expect the lane to do it's thing without you having to look at it, it might happen that it'll sit there waiting for input for a while, possibly delaying the build going out

⚖️ 

<details>
 <summary>My opinion</summary>
 While the "run and forget" option is convenient, as long as we have release steps that need to be initiated by a human on their machine, I think it's appropriate for said human to also pay close attention to what the lane does. As such, I'm leaning towards keeping the blocking request for user input. 
</details>